tool/templates/dart/init.dart

```diff
@@ -303,4 +303,11 @@ class _FinalizedArena {
   _FinalizedArena() : this.arena = ffi2.Arena() {
     _finalizer.attach(this, this.arena);
   }
+  // Construct
+  _FinalizedArena.withLifetime(core.List<core.List<Object>> lifetimeAppendArray) : this.arena = ffi2.Arena() {
+    _finalizer.attach(this, this.arena);
```

Comment:
you can slightly simplify this by invoking the main constructor

```suggestion
  _FinalizedArena.withLifetime(core.List<core.List<Object>> lifetimeAppendArray) : this() {
```

Reply:
Nope:

```
  lib/src/lib.g.dart:348:91: Error: Redirecting constructors can't have a body.
  Try removing the body, or not making this a redirecting constructor.
    _FinalizedArena.withLifetime(core.List<core.List<Object>> lifetimeAppendArray) : this() {
```